src/components/timeline/TrackCustomMarkerGraph.tsx

```diff
@@ -336,6 +338,7 @@ type StateProps = {
   readonly rangeEnd: Milliseconds;
   readonly markerSampleRanges: [IndexIntoSamplesTable, IndexIntoSamplesTable];
   readonly collectedSamples: CollectedCustomMarkerSamples;
+  readonly minMax: { minNumber: number; maxNumber: number };
```

Comment:
This type is redeclared three times in this diff, I think it's worth defining it globally. `Bounds` or `MarkerSampleBounds` or `ValueBounds` could work as a type name.